bgpd/bgp_advertise.c

```diff
@@ -181,7 +181,7 @@ bool bgp_adj_out_lookup(struct peer *peer, struct bgp_dest *dest,
 
 
 void bgp_adj_in_set(struct bgp_dest *dest, struct peer *peer, struct attr *attr,
-		    uint32_t addpath_id)
+		    uint32_t addpath_id, mpls_label_t *label, uint32_t num_labels)
```

Comment:
Why is this data not stored in the attr?

Reply:
If this is a suggestion, this can be added.
Just be aware the sizing impact (attr would be added 12 extra bytes => 10MB for a full route).